Makefile

```diff
@@ -225,10 +225,13 @@ mrsigner:
 
 .PHONY: update
 update:
+
 	@echo "Running cargo update.."
 	@cargo update
 	@echo "cargo update in enclave directory"
 	@cd enclave && cargo update
+	@cd enclave && cargo update -p jsonrpc-core --precise e5ee60bc30dedf513743843be2523dc384bbcae1
```

Comment:
why is there a duplicate?

Reply:
To ensure it has updated & removed all obsolete crates correctly.

Reply:
ok